src/model/semantics.ml

```diff
@@ -216,23 +216,39 @@ let rec inline_element :
 
 and inline_elements status elements = List.map (inline_element status) elements
 
+let generate_label =
+  let current_label = ref 0 in
+  fun status ->
+    incr current_label;
+    Paths.Identifier.Mk.label
+      ( status.parent_of_sections,
+        Names.LabelName.make_std ("search_label_" ^ string_of_int !current_label)
```

Comment:
That's not a very nice label. People will start to link to these things so they will show up in URLs. Can we maybe find a shorter scheme and perhaps give a better hint at the kind of element we are linking to ? Maybe something like `p%d`, `code%d`, `math%d` or whatever better scheme you can come up with.